src/init.cpp

```diff
@@ -992,6 +975,13 @@ void InitParameterInteraction()
         gArgs.ForceSetArg("-checklevel", "4");
         LogPrintf("%s: parameter interaction: additional indexes -> setting -checklevel=4\n", __func__);
     }
+
+    if (gArgs.IsArgSet("-masternodeblsprivkey")) {
+        if (gArgs.SoftSetArg("-maxconnections", itostr(DEFAULT_MAX_PEER_CONNECTIONS))) {
```

Comment:
isn't masternode max peer count supposed to be higher than normal peers?

Reply:
No, but it supposed to be _at least_ `DEFAULT_MAX_PEER_CONNECTIONS` while normal nodes might reduce it. Anyway, this code is actually a redundant one 🙈 removed it.